marlowe-runtime/cli/Language/Marlowe/Runtime/CLI/Command/Create.hs

```diff
@@ -0,0 +1,84 @@
+module Language.Marlowe.Runtime.CLI.Command.Create
+  where
+
+import Data.Map (Map)
+import qualified Data.Map as Map
+import Data.String (fromString)
+import Language.Marlowe (POSIXTime(POSIXTime))
+import Language.Marlowe.Runtime.CLI.Command.Tx (TxCommand, txCommandParser)
+import Language.Marlowe.Runtime.CLI.Monad (CLI)
+import Language.Marlowe.Runtime.CLI.Option (keyValueOption, marloweVersionParser, parseAddress)
+import Language.Marlowe.Runtime.ChainSync.Api (Address, TokenName(..))
+import Language.Marlowe.Runtime.Core.Api (SomeMarloweVersion)
+import Options.Applicative
+import Text.Read (readMaybe)
+
+data CreateCommand = CreateCommand
+  { marloweVersion :: SomeMarloweVersion
+  , roles :: Map TokenName Address
```

Comment:
If `create` is minting the role tokens, then we need to let the user provide CIP-25 metadata (key 721) for the role tokens.

Reply:
It seems like there is a larger conversation about minting and role tokens that needs to happen